BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -128,16 +128,22 @@ public struct AttributedLabel: Element, Hashable {
     /// You can check if this value should be false via `NSAttributedString.needsNormalizingForView(...)`
     public var needsTextNormalization: Bool = true
 
-    private static let prototypeLabel = LabelView()
```

Comment:
@watt suggested we consider leaving this one as-is, at least for the initial iteration since its:
- low likelihood of retaining something it shouldn't be (and would be flushed out by other measurements)
- on a relatively hot code path